File: src/main/java/org/jesperancinha/rockstarts/rockstarsmanager/data/ArtistDto.java
package org.jesperancinha.rockstarts.rockstarsmanager.data;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.persistence.Id;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ArtistDto {

    @Id
    @JsonProperty("Id")
    private Long id;

    @JsonProperty("Name")
    private String name;
}

File: src/main/kotlin/org/jesperancinha/rockstarts/rockstarsmanager/data/ArtistDto.kt
package org.jesperancinha.rockstarts.rockstarsmanager.data

import com.fasterxml.jackson.annotation.JsonProperty
import jakarta.persistence.Id
import lombok.*

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
class ArtistDto {
    @Id
    @JsonProperty("Id")
    private val id: Long? = null

    @JsonProperty("Name")
    private val name: String? = null
}